docs/tutorials/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>Press <a href="../monitoring/model-monitoring-deployment.html#model-endpoint-summary-list" target="_blank">Features Analysis</a> to see details of the drift analysis in a table format with each feature in the selected model on its own line, including the predicted label:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>